lib/stats/DefaultStatsFactoryPlugin.js

```diff
@@ -1779,6 +1811,60 @@ const spaceLimited = (
 	};
 };
 
+const errorsSpaceLimit = (errors, max) => {
+	let filtered = 0;
+	// Can not fit into limit
+	// print only messages
+	if (errors.length >= max)
+		return [
+			errors.map(error => {
+				if (error.details === undefined) return error;
+				filtered++;
+				return { ...error, details: "" };
+			}),
+			filtered
+		];
+	let fullLength = errors.length;
+	let result = errors;
+
+	let i = 0;
+	for (; i < errors.length; i++) {
+		if (errors[i].details) {
+			const splitted = errors[i].details.split("\n");
+			const len = splitted.length;
+			fullLength += len;
+			if (fullLength > max) {
+				result = i > 0 ? errors.slice(0, i) : [];
+				const overlimit = fullLength - max;
+				result.push({
+					...errors[i++],
+					details: `${splitted
+						.slice(0, len - overlimit)
+						.join("\n")}\n+${overlimit} hidden line${
+						overlimit === 1 ? "" : "s"
+					}`
```

Comment:
The extra line should be added by the `DefaultStatsPrinterPlugin`.
Only add a `filteredDetails` property to the object instead.

Reply:
this lines is related to partial details. e.g. case 2 errors.
- first has 3 lines of details
- second 2 lines.
- errorsScope=5
then we should get:
<error #1 message> (1)
<error #1 details line #1> (2)
<error #1 details line #2> (3)
+ 1 hidden line (4)

<error #2 message> (5)